collections/includes_value.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Returns whether the given value is part of the collection.
+ *
+ * Example:
+ * ```ts
+ * import { includesValue } from "./includes_value.ts";
+ * import { assert } from "../testing/asserts.ts";
+ *
+ * const input = {
+ *   first: 33,
+ *   second: 34,
+ * };
+ *
+ * assert(includesValue(input, 34));
+ */
+
+export function includesValue<T>(
+  collection: Record<string, T>,
+  value: T,
+): boolean {
+  return Object.values(collection).includes(value);
```

Comment:
We should be better on memory and runtime than that, as `Object.values` needs to get **all** values and needs to allocate a whole array of all of them, just to do an `.includes` here. You should be able to do that by hand with `O(1)` memory and the same or better runtime.